### ISSUE TITLE
Keyboard: the configure button in the notification pop is not accessible by tab key

### ISSUE THREAD

**Issue by fsteffi** [microsoft/vscode#184711]
**GitHub Tags**:#A11yTCS;#A11y_VSCode;#CAITestPass_May2020; #DesktopApp;#Win11;#MAS2.4.3;#Benchmark;#FTP;#A11yMAS;#A11ySev2;#AINotInScope;#Keyboard;#Visual Studio Code;

## Environment Details:
Application Name: Azure Extensions for VS Code
Version: 1.80.0-insider (user setup)
Commit: db6bcdf309cc1565c502a54b73c33a63bdabfbd8
Date: 2023-06-09T05:21:13.157Z
Electron: 22.5.5
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.25370
Edition	Windows 11 Enterprise
Version	22H2
Installed on	‎29-‎05-‎2023
OS build	25370.1
Experience	Windows Feature Experience Pack 1000.25350.1000.0

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification.
2. Open Visual studio code
3. Click on extension button and search for "A11y-kit"
4. Click on install 
5.  run F1 > a11y-kit: Select UI feature for accessibility testing.
6. click show information messages
8. Navigate to configure notification button and expand and then collapse with the help of esc key.

## Actual:
the configure button the notification is not accessible with tab key.

## Expected:
Focus should move to the configure button on tab navigation

## User Impact:
User will not be able to open the configure button

## Attachment:
[Video 6-09 at 15.49.webm](https://github.com/microsoft/vscode/assets/24580570/d7aef8e3-5183-4a86-a6c2-215a4a29439d)

**Comment by fsteffi** [comment]
logged in ref to bug https://github.com/Microsoft/vscode-azurefunctions/issues/3637

**Comment by bpasero** [comment]
There is no configure button in all notifications only those that have configure actions. I think for these the button is always visible:

![image](https://github.com/microsoft/vscode/assets/900690/3e55b84b-20d9-4821-ae89-2f2737e7e8ae)

What are the exact steps to reproduce here?

**Comment by fsteffi** [comment]
HI @bpasero  the button is not always visible hence the issue.

1. Open Visual studio code
2. Click on extension button and search for "A11y-kit"
3. Click on install
4. run F1 > a11y-kit: Select UI feature for accessibility testing.
5. click show information messages
6. Navigate to configure notification button and expand and then collapse with the help of esc key.

**Comment by bpasero** [comment]
This is a great finding and I even figured out an issue with notifications in general. The actions in the toolbar are not meant to always show, only when the toast has focus.

**Comment by fsteffi** [comment]
#closed;
[Video 6-14 at 16.15.webm](https://github.com/microsoft/vscode/assets/24580570/03134505-bd5c-430b-b039-b9427967651c)

Version: 1.80.0-insider (user setup)
Commit: f3aa9a201b2455e6556797e8ffb7145d77adb792
Date: 2023-06-14T09:02:23.566Z
Electron: 22.5.7
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.25375
